### ISSUE TITLE
No help message for `rustup-init --help`

### ISSUE THREAD

**Issue by tesuji** [rust-lang/rustup#2395]
**Problem**

No help message for `rustup-init --help` or `-h`

**Notes**

Commit: 138d33e501899946bdab38c7af161d6ec24c1601
Output of `rustup show`:
```
rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/lzutao/.rustup

installed toolchains
--------------------

nightly-x86_64-unknown-linux-gnu
master
stage1

active toolchain
----------------

nightly-x86_64-unknown-linux-gnu (default)
rustc 1.46.0-nightly (f844ea1e5 2020-07-03)
```

**Comment by kinnison** [comment]
This is a known issue we have a fix already on the stable branch which will be merged as part of the 1.22 release.

**Comment by kinnison** [comment]
1.22.0 was released, and this fix as part of it, sorry for the delays.
